### ISSUE TITLE
Warnings when building Xen bindings on Ubuntu 19.10

### ISSUE THREAD

**Issue by samoht** [Solo5/solo5#475]
I've got:

```
$ uname -a
Linux porto 5.3.0-55-generic #49-Ubuntu SMP Thu May 21 12:47:19 UTC 2020 x86_64 x86_64 x86_64 GNU/Linux
$ lsb_release -a
No LSB modules are available.
Distributor ID:	Ubuntu
Description:	Ubuntu 19.10
Release:	19.10
Codename:	eoan
$ ./configure && make V=1 CONFIG_HVT= CONFIG_SPT= CONFIG_VIRTIO= CONFIG_XEN=1 CONFIG_MUEN=
xen/hypercall_page.S: Assembler messages:
xen/hypercall_page.S:85: Warning: symbol 'HYPERCALL_set_trap_table' already has its type set
xen/hypercall_page.S:86: Warning: symbol 'HYPERCALL_mmu_update' already has its type set
xen/hypercall_page.S:87: Warning: symbol 'HYPERCALL_set_gdt' already has its type set
xen/hypercall_page.S:88: Warning: symbol 'HYPERCALL_stack_switch' already has its type set
xen/hypercall_page.S:89: Warning: symbol 'HYPERCALL_set_callbacks' already has its type set
xen/hypercall_page.S:90: Warning: symbol 'HYPERCALL_fpu_taskswitch' already has its type set
xen/hypercall_page.S:91: Warning: symbol 'HYPERCALL_sched_op_compat' already has its type set
xen/hypercall_page.S:92: Warning: symbol 'HYPERCALL_platform_op' already has its type set
xen/hypercall_page.S:93: Warning: symbol 'HYPERCALL_set_debugreg' already has its type set
xen/hypercall_page.S:94: Warning: symbol 'HYPERCALL_get_debugreg' already has its type set
xen/hypercall_page.S:95: Warning: symbol 'HYPERCALL_update_descriptor' already has its type set
xen/hypercall_page.S:96: Warning: symbol 'HYPERCALL_memory_op' already has its type set
xen/hypercall_page.S:97: Warning: symbol 'HYPERCALL_multicall' already has its type set
xen/hypercall_page.S:98: Warning: symbol 'HYPERCALL_update_va_mapping' already has its type set
xen/hypercall_page.S:99: Warning: symbol 'HYPERCALL_set_timer_op' already has its type set
xen/hypercall_page.S:100: Warning: symbol 'HYPERCALL_event_channel_op_compat' already has its type set
xen/hypercall_page.S:101: Warning: symbol 'HYPERCALL_xen_version' already has its type set
xen/hypercall_page.S:102: Warning: symbol 'HYPERCALL_console_io' already has its type set
xen/hypercall_page.S:103: Warning: symbol 'HYPERCALL_physdev_op_compat' already has its type set
xen/hypercall_page.S:104: Warning: symbol 'HYPERCALL_grant_table_op' already has its type set
xen/hypercall_page.S:105: Warning: symbol 'HYPERCALL_vm_assist' already has its type set
xen/hypercall_page.S:106: Warning: symbol 'HYPERCALL_update_va_mapping_otherdomain' already has its type set
xen/hypercall_page.S:107: Warning: symbol 'HYPERCALL_iret' already has its type set
xen/hypercall_page.S:108: Warning: symbol 'HYPERCALL_vcpu_op' already has its type set
xen/hypercall_page.S:109: Warning: symbol 'HYPERCALL_set_segment_base' already has its type set
xen/hypercall_page.S:110: Warning: symbol 'HYPERCALL_mmuext_op' already has its type set
xen/hypercall_page.S:111: Warning: symbol 'HYPERCALL_xsm_op' already has its type set
xen/hypercall_page.S:112: Warning: symbol 'HYPERCALL_nmi_op' already has its type set
xen/hypercall_page.S:113: Warning: symbol 'HYPERCALL_sched_op' already has its type set
xen/hypercall_page.S:114: Warning: symbol 'HYPERCALL_callback_op' already has its type set
xen/hypercall_page.S:115: Warning: symbol 'HYPERCALL_xenoprof_op' already has its type set
xen/hypercall_page.S:116: Warning: symbol 'HYPERCALL_event_channel_op' already has its type set
xen/hypercall_page.S:117: Warning: symbol 'HYPERCALL_physdev_op' already has its type set
xen/hypercall_page.S:118: Warning: symbol 'HYPERCALL_hvm_op' already has its type set
xen/hypercall_page.S:119: Warning: symbol 'HYPERCALL_sysctl' already has its type set
xen/hypercall_page.S:120: Warning: symbol 'HYPERCALL_domctl' already has its type set
xen/hypercall_page.S:121: Warning: symbol 'HYPERCALL_kexec_op' already has its type set
xen/hypercall_page.S:122: Warning: symbol 'HYPERCALL_tmem_op' already has its type set
xen/hypercall_page.S:124: Warning: symbol 'HYPERCALL_xenpmu_op' already has its type set
xen/hypercall_page.S:126: Warning: symbol 'HYPERCALL_arch_0' already has its type set
xen/hypercall_page.S:127: Warning: symbol 'HYPERCALL_arch_1' already has its type set
xen/hypercall_page.S:128: Warning: symbol 'HYPERCALL_arch_2' already has its type set
xen/hypercall_page.S:129: Warning: symbol 'HYPERCALL_arch_3' already has its type set
xen/hypercall_page.S:130: Warning: symbol 'HYPERCALL_arch_4' already has its type set
xen/hypercall_page.S:131: Warning: symbol 'HYPERCALL_arch_5' already has its type set
xen/hypercall_page.S:132: Warning: symbol 'HYPERCALL_arch_6' already has its type set
xen/hypercall_page.S:133: Warning: symbol 'HYPERCALL_arch_7' already has its type set
```

Is this expected?

**Comment by mato** [comment]
Thanks for reporting, the warning is harmless but annoying; I have a fix in #476, just waiting for the CI to pass.

**Comment by mato** [comment]
Fixed in #476.
